### ISSUE TITLE
Undefined error on option2.rs

### ISSUE THREAD

**Issue by Descent098** [rust-lang/rustlings#550]
I was working on ```option2.rs``` and ran into an undefined error:

![image](https://user-images.githubusercontent.com/22761705/95038157-b9f49700-068a-11eb-8d4f-86ec3f87adab.png)

The code I used was:

```rust
// option2.rs
// Make me compile! Execute `rustlings hint option2` for hints


fn main() {
    let optional_value = Some(String::from("rustlings"));
    // TODO: Make this an if let statement whose value is "Some" type
    if let value = optional_value {
        println!("the value of optional value is: {}", value.unwrap());
    } else {
        println!("The optional value doesn't contain anything!");
    }

    let mut optional_values_vec: Vec<Option<i8>> = Vec::new();
    for x in 1..10 {
        optional_values_vec.push(Some(x));
    }

    // TODO: make this a while let statement - remember that vector.pop also adds another layer of Option<T>
    // You can stack `Option<T>`'s into while let and if let
    while let value = optional_values_vec.pop() {
        println!("current value: {}", value.unwrap().unwrap());
    }
}
```
I don't know enough to be able to debug what I did wrong here, but whatever it was should probably not fail silently. Let me know if any additional details are needed Thanks!

## Details
```rustc --version``` == ```rustc 1.46.0 (04488afe3 2020-08-24)```
```rustlings --version ```== ```rustlings 4.0.0```
```ls -la``` == Not available on windows, issue occured inside ```exercises/option/option2.rs```
Your OS name and version == ```Windows 10 pro 10.0.19041```

**Comment by jrvidal** [comment]
This is a duplicate of #424. It is fixed in `main` but we haven't released a new version yet, sorry :confused:

**Comment by Descent098** [comment]
@jrvidal Ah, perfect thanks!

**Comment by fmoko** [comment]
Just released 4.1 which should include this fix.
